WcaOnRails/db/migrate/20161117085757_cleanup_events_issue96.rb

```diff
@@ -0,0 +1,12 @@
+class CleanupEventsIssue96 < ActiveRecord::Migration
+  def up
+    remove_foreign_key :preferred_formats, column: :event_id
+    remove_foreign_key :competition_events, column: :event_id
```

Comment:
The simplest way is to run the SQL statement `SET FOREIGN_KEY_CHECKS=0;` followed by your code to delete and then reload followed by another SQL statement `SET FOREIGN_KEY_CHECKS=1;`